cg/services/order_validation_service/validators/inter_field/rules.py

```diff
@@ -70,3 +71,13 @@ def validate_buffers_are_allowed(order: TomteOrder) -> list[CaseSampleError]:
                 error = InvalidBufferError(case_name=case.name, sample_name=sample.name)
                 errors.append(error)
     return errors
+
+
+def validate_subject_ids_different_from_sample_names(order: TomteOrder) -> list[CaseSampleError]:
+    errors = []
+    for case in order.cases:
```

Comment:
Consider adding a method to the `TomteOrder` class called `get_samples` to get rid of these loops here 😄

Reply:
I like it! Would you prefer that as a method or a property? order.get_samples() vs order.samples

Reply:
Also, I need both the case name and the sample name

Reply:
Ah, true... Do we want to map in the case name to the samples? I feel like it would make our lives easier :)

Reply:
Not possible since a sample can be in multiple cases 😅 Lets leave it